jenkins/shared-library/vars/monorepoSeedPipeline.groovy

```diff
@@ -94,5 +94,5 @@ void call(Closure body) {
 
 String resolveOwnerAndProject(String gitUrl) {
 	String[] tokens = gitUrl.tokenize('/')
-	return "${tokens[2]}/${tokens.last().split('\\.')[0]}"
+	return [tokens[2], tokens.last().split('\\.')[0]]
```

Comment:
why this change?

Reply:
the git url was getting parse twice, once in this method and another above.
With this change the git url only get parse once and the required two parts are returned to the caller.